### PR TITLE
migrated oembed

### PR DESCRIPTION
**Pull request by javisantana**

### CODE REVIEW

**Comment by javisantana** [comment]
cc @Kartones

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3922/))

**Comment by Kartones** [comment]
Probably the build will fail because you need to move this specs (or at least make them call the new controller): https://github.com/CartoDB/cartodb/blob/master/spec/unit/controllers/api/json/oembed_controller_spec.rb

File: app/controllers/carto/api/oembed_controller.rb [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+        raise ActionController::RoutingError.new('Incorrect width') if (width =~ /\A[0-9]+(%|px)?\z/).nil?
+        raise ActionController::RoutingError.new('Incorrect height') if (height =~ /\A[0-9]+(%|px)?\z/).nil?
+
+        uri = URI.parse(url)
+
+        uuid = extract_uuid_from_string(uri.path)
+        raise ActionController::RoutingError.new('UUID not found in URL') if uuid.nil?
+
+        vis = Visualization.where(id: @id).first
```
Comment (Kartones) [inline app/controllers/carto/api/oembed_controller.rb]: Missing namespace, please to be consisten use `Carto::Visualization`

**Comment by Kartones** [comment]
Apart from that, :+1:

**Comment by javisantana** [comment]
ok, will move, thanks

File: app/controllers/carto/api/oembed_controller.rb (hunk 1 above)
Comment (javisantana) [inline app/controllers/carto/api/oembed_controller.rb]: :es:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3948/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3950/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3952/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3959/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4005/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4103/))

**Pull request merged**